### PR TITLE
Docs(web-twig, web-react): Improve `Modal` documentation regarding `ScrollView` and optional title

### PR DESCRIPTION
**Pull request by adamkudrna**
https://jira.lmc.cz/browse/DS-876

### CODE REVIEW

File: packages/web-twig/src/Resources/components/Modal/README.md [hunk 1]
```diff
@@ -186,6 +195,21 @@ further extend component's descriptiveness and accessibility. Also, UNSAFE styli
...
 
+## Scrolling Long Content
+
+When Modals become too long for the user's viewport or device, they automatically scroll independent of the page itself.
+
+To make content overflow more obvious to users, you can wrap the ModalBody content in a [ScrollView][scroll-view] to
+take over the responsibility for scrolling, e.g.:
+
+```twig
+<ScrollView data-spirit-toggle="scrollView" indicators="both">
```
Comment (crishpeen) [inline packages/web-twig/src/Resources/components/Modal/README.md:206]:
```suggestion
<ScrollView data-spirit-toggle="scrollView" overflowDecorators="both">
```

File: packages/web-twig/src/Resources/components/Modal/README.md [hunk 2]
```diff
@@ -213,9 +237,11 @@ When you put it all together:
     >
       Title of the Modal
     </ModalHeader>
-    <ModalBody>
-      <p>Modal body form</p>
-    </ModalBody>
+    <ScrollView data-spirit-toggle="scrollView" indicators="both">
```
Comment (crishpeen) [inline packages/web-twig/src/Resources/components/Modal/README.md:240]:
```suggestion
    <ScrollView data-spirit-toggle="scrollView" overflowDecorators="both">
```

File: packages/web-twig/src/Resources/components/Modal/stories/ModalScrollingLongContent.twig [hunk 3]
```diff
@@ -84,7 +84,7 @@
             <ModalHeader modalId="example_scroll_view" titleId="example_scroll_view_title">
                 Modal with ScrollView
             </ModalHeader>
-            <ScrollView data-spirit-toggle="scrollView" indicators="borders">
+            <ScrollView data-spirit-toggle="scrollView" indicators="both">
```
Comment (crishpeen) [inline packages/web-twig/src/Resources/components/Modal/stories/ModalScrollingLongContent.twig:87]:
```suggestion
            <ScrollView data-spirit-toggle="scrollView" overflowDecorators="both">
```

**Review by crishpeen — CHANGES_REQUESTED** [review]

**Review by crishpeen — APPROVED** [review]

**Review by pavelklibani — APPROVED** [review]

File: packages/web-react/src/components/Modal/README.md [hunk 4]
```diff
@@ -112,9 +129,23 @@ The rest of the properties are created from the default `<div>` element. [Docs][
...
+## Scrolling Long Content
+
+When Modals become too long for the user's viewport or device, they automatically scroll independent of the page itself.
+
+To make content overflow more obvious to users, you can wrap the ModalBody content in a [ScrollView][scroll-view] that
+takes over the responsibility for scrolling and provides visual overflow decorators, e.g.:
+
+```jsx
+<ScrollView overflowDecorators="both">
+  <ModalBody>…Long content…</ModalBody>
```
Comment (dlouhak) [inline packages/web-react/src/components/Modal/README.md:141]: ?

```suggestion
  <ModalBody>
      …Long content…
  </ModalBody>
```

File: packages/web-react/src/components/Modal/README.md [hunk 5]
```diff
@@ -112,9 +129,23 @@ The rest of the properties are created from the default `<div>` element. [Docs][
 
 The rest of the properties are based from type of `HTMLElement`. [Docs][HTMLElementDocs]
 
+## Scrolling Long Content
+
+When Modals become too long for the user's viewport or device, they automatically scroll independent of the page itself.
+
+To make content overflow more obvious to users, you can wrap the ModalBody content in a [ScrollView][scroll-view] that
```
Comment (dlouhak) [inline packages/web-react/src/components/Modal/README.md:136]: ? 

```suggestion
To make content overflow more obvious to users, you can wrap the `ModalBody` content in a [ScrollView][scroll-view] that
```

File: packages/web-react/src/components/Modal/README.md (hunk 4 above)
Comment (literat) [reply packages/web-react/src/components/Modal/README.md:141]: Fixed.

**Pull request merged**